src/main/java/org/mvplugins/multiverse/core/commandtools/MVCommandCompletions.java

```diff
@@ -152,4 +160,28 @@ private List<String> getMVWorldNames(BukkitCommandCompletionContext context) {
         Logging.severe("Invalid MVWorld scope: " + scope);
         return Collections.emptyList();
     }
+
+    private Collection<String> suggestMVWorldPropsName(BukkitCommandCompletionContext context) {
+        return Try.of(() -> {
+            MultiverseWorld mvWorld = context.getContextValue(MultiverseWorld.class);
+            ConfigModifyType modifyType = context.getContextValue(ConfigModifyType.class);
+            return mvWorld.getConfigurablePropertyNames(modifyType);
+        }).getOrElse(Collections.emptyList());
+    }
+
+    private Collection<String> suggestMVWorldPropsValue(BukkitCommandCompletionContext context) {
+        return Try.of(() -> {
+            MultiverseWorld mvWorld = context.getContextValue(MultiverseWorld.class);
+            ConfigModifyType modifyType = context.getContextValue(ConfigModifyType.class);
+            String propertyName = context.getContextValue(String.class);
+            return mvWorld.suggestPropertyValues(modifyType, propertyName, context.getInput());
```

Comment:
Why is a world suggesting property values? This smells like MultiverseWorld is doing too much.